debian/mgmt.service

```diff
@@ -0,0 +1 @@
+../misc/mgmt.service
```

Comment:
Still it should belong to install...

«If your service file is not provided by upstream and you ship it as part of your debian package, name it debian/$pkg.service and dh_installinit will install it to the correct location. For more information please refer to the dh_installinit(1) man page. »

In this case it is provided by mgmt, so install is the place.